rtmp-client/src/main/cpp/librtmp/rtmp.h

```diff
@@ -102,6 +102,40 @@ extern "C"
 #define RTMP_PACKET_SIZE_SMALL    2
 #define RTMP_PACKET_SIZE_MINIMUM  3
 
+  typedef enum RTMPErrors_ {
+    RTMP_SUCCESS = 1,
+    RTMP_ERROR_NONE = 0,
```

Comment:
Why do we need `RTMP_SUCCESS` and `RTMP_ERROR_NONE` as separate values? I would have thought "no error" and "success" mean the same thing. Can we use `RTMP_SUCCESS = 0` everywhere?

Reply:
Done.